components/alert/AlertMap.vue

```diff
@@ -327,365 +402,446 @@ export default Vue.extend({
-          link?.setStyle({
+        else if (
+          region.facilityType === FacilityType.WARPGATE ||
+          connection.facilityType === FacilityType.WARPGATE
+        ) {
+          link.setStyle({
             color: MAP_LINK_COLORS[0].toString(),
-            dashArray: "5 5"
-          });
-          bglink?.setStyle({
+            dashArray: '5 5',
+          })
+          bglink.setStyle({
+            stroke: true,
             color: MAP_LINK_COLORS[6].toString(),
             opacity: MAP_LINK_COLORS[6].a,
-            dashArray: "5 5",
-            dashOffset: "5"
-          });
+            dashArray: '5 5',
+            dashOffset: '5',
+          })
         }
         // Enemy factions, set capturable color, dashes, and enable bglink
         else {
-          link?.setStyle({
+          link.setStyle({
             color: MAP_LINK_COLORS[4].toString(),
-            dashArray: "5 5"
-          });
-          bglink?.setStyle({
+            dashArray: '5 5',
+          })
+          bglink.setStyle({
+            stroke: true,
             color: MAP_LINK_COLORS[5].toString(),
             opacity: MAP_LINK_COLORS[5].a,
-            dashArray: "5 5",
-            dashOffset: "5"
-          });
+            dashArray: '5 5',
+            dashOffset: '5',
+          })
         }
-      });
+      })
     },
     async pull(): Promise<void> {
-        if (this.loaded && this.alert.state === Ps2alertsEventState.ENDED) {
-          return
-        }
+      if (this.loaded && this.alert.state === Ps2alertsEventState.ENDED) {
+        return
+      }
+
+      this.outfitData = await this.pullOutfitData(
+        this.alert.instanceId ?? '12345'
+      )
 
-        this.outfitData = await this.pullOutfitData(
-          this.alert.instanceId ?? "12345"
+      console.log('AlertMap.pull', this.alert.instanceId)
+      await new ApiRequest()
+        .get<InstanceFacilityControlEntriesResponseInterface[]>(
+          Endpoints.INSTANCE_FACILITY_CONTROL_ENTRIES.replace(
+            '{instance}',
+            this.alert.instanceId
+              ? this.alert.instanceId.toString()
+              : 'whatever'
+          )
         )
-      
-        console.log('AlertMap.pull', this.alert.instanceId)
-        await new ApiRequest()
-          .get<InstanceFacilityControlEntriesResponseInterface[]>(
-            Endpoints.INSTANCE_FACILITY_CONTROL_ENTRIES.replace(
-              '{instance}',
-              this.alert.instanceId
-                ? this.alert.instanceId.toString()
-                : 'whatever'
+        .then((result) => {
+          // Copy the history since updateTerritory reverses the provided list
+          this.historyCache = Object.assign([], result)
+          const capture = this.updateTerritory(result, undefined)
+          if (capture) {
+            this.updateCutoffs()
+          }
+          this.loaded = true
+          const now = new Date()
+          this.lastUpdated = new Date(
```

Comment:
I was thinking "surely there must be a way to get just a new UTC date without having to define the `.UTC()` constructor..."

Turns out I was wrong. Wow JavaScript... wow.

Reply:
I was pretty amazed as well lol